shell_integration/nautilus/syncstate.py

```diff
@@ -97,9 +101,11 @@ def _connectToSocketServer(self):
             self._sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
             sock_file = os.path.join(get_runtime_dir(), appname, "socket")
             try:
-                self._sock.connect(sock_file) # fails if sock_file doesn't exist
+                self._sock.connect(sock_file)  # fails if sock_file doesn't exist
```

Comment:
did `black` make this change?

Reply:
maybe

Reply:
I must revert them back. They are source code files.